### PR TITLE
Change gear link to lead to edit_project page

### PR DESCRIPTION
**Pull request by Lerek**

### CODE REVIEW

**Review by Riddlerrr — APPROVED** [review]

**Review by aquabu — CHANGES_REQUESTED** [review]
Please add an integration spec.

**Comment by Lerek** [comment]
> Please add an integration spec.

@aquabu done

File: spec/features/project_discovery/landing_page_spec.rb [hunk 1]
```diff
@@ -32,6 +32,10 @@
 
     expect(page.all('.project').size).to eq(16)
 
+    page.find_all('a.fa.fa-cog').each do |node|
```
Comment (Riddlerrr) [inline spec/features/project_discovery/landing_page_spec.rb:35]: Do we have multiple projects in this example or just one?

**Review by Riddlerrr — APPROVED** [review]

File: spec/features/project_discovery/landing_page_spec.rb (hunk 1 above)
Comment (Lerek) [reply spec/features/project_discovery/landing_page_spec.rb:35]: @Riddlerrr there are 14 projects :)

**Review by aquabu — APPROVED** [review]

**Pull request merged**